runtime/jvmti/jvmtiHook.c

```diff
@@ -1582,20 +1582,24 @@ jvmtiHookGetEnv(J9HookInterface** hook, UDATA eventNum, void* eventData, void* u
 	if (data->rc == JNI_EVERSION) {
 		jint version = data->version & ~JVMTI_VERSION_MASK_MICRO;
 
-		if ((version == JVMTI_VERSION_1_0) 
-			|| (version == JVMTI_VERSION_1_1) 
-			|| (version == JVMTI_VERSION_1_2) 
-			|| (version == JVMTI_VERSION_9)
-			|| (version == JVMTI_VERSION_11)
+		if ((JVMTI_VERSION_1_0 == version)
+			|| (JVMTI_VERSION_1_1 == version)
+			|| (JVMTI_VERSION_1_2 == version)
+#if JAVA_SPEC_VERSION > 8
+			|| (JVMTI_VERSION_9 == version)
+#if JAVA_SPEC_VERSION >= 11
```

Comment:
To be consistent with jvmti.h.m4, this check could be removed.

Reply:
Agreed, removed it.